### ISSUE TITLE
Search text style

### ISSUE THREAD

**Issue by jalal-zy** [Daniel-Ioannou/flutter_country_picker#92]
Please add 'style' to the text search field 
So I can add custom style to the search text field (color, font)

**Comment by ewankirk** [comment]
+1.  I can customize everything but the textStyle of the search box is fixed.  If you set the background color of the dropdown sheet as a dark color then there's no way to change the color of the text to a contrasting one.

**Comment by ewankirk** [comment]
Fixed this and created a PR for it.

**Comment by Daniel-Ioannou** [comment]
@ewankirk Thank you for your contribution. Released in [v2.0.17](https://pub.dev/packages/country_picker/versions/2.0.17).

**Comment by Technorocker** [comment]
I just found this plugin and I am unable to style the search text or border. I am able to change the color of the prefix icon though.

here is my function when I trigger it
```
   showCountryPicker(
        context: context,
        favorite: <String>['CA', 'US'],
        countryListTheme: CountryListThemeData(
          flagSize: 30,
          backgroundColor: kPrimaryColor,
          textStyle: const TextStyle(fontSize: 20, color: kSecondaryColor),
          searchTextStyle: const TextStyle(fontSize: 20, color: Colors.white),
          bottomSheetHeight: 500,
          borderRadius: const BorderRadius.only(
            topLeft: Radius.circular(20.0),
            topRight: Radius.circular(20.0),
          ),
          inputDecoration: InputDecoration(
            labelText: 'Search',
            hintText: 'Start typing to search',
            prefixIcon: const Icon(Icons.search),
            prefixIconColor: kSecondaryColor,
            border: OutlineInputBorder(
              borderRadius: BorderRadius.circular(20),
              borderSide: BorderSide(
                color: kSecondaryColor.withOpacity(0.2),
              ),
            ),
          ),
        ),
        onSelect: (Country country) {
          setState(() {
            countryCode = country;
          });
        });
```
